en/installation.md

```diff
@@ -289,6 +289,18 @@ extension=php_phalcon.so
 
 Restart your webserver.
 
+### PHPBrew (macOS/Linux)
+
```

Comment:
Can you add line that mentions you need to have PHPBrew installed with a link to the installation instructions? (link in bottom of file)